Automattic/wc-calypso-bridge
# Add the Plugins menu item back to free trial sites

### Specification

This PR adds a new "Plugins" landing page for WX trial sites at the `domain.com/wp-admin/admin.php?page=plugins-upgrade` URL.

It also introduces the following track events:
- `free_trial_upgrade_now`
- `free_trial_browse_plugins`
- `free_trial_discover_extensions`
- `free_trial_get_inspired`

**Note:** Every track includes the `source: plugins` metadata.

Closes #1191 

### Validation

- Visit a WX trial site. 
- Ensure you see the "Plugins" landing page in the admin menu (and the expected priority.)
- Ensure the landing page doesn't appear for other WX plans (Essential, Performance) and the WPCOM's ecommerce plan.
